File: problem-1/author-solutions/s.py
s = input()
a = e = i = o = u = False
for c in s:
  if not a and c.lower() == 'a':
    a = True
  if not e and c.lower() == 'e':
    e = True
  if not i and c.lower() == 'i':
    i = True
  if not o and c.lower() == 'o':
    o = True
  if not u and c.lower() == 'u':
    u = True
print("Yes" if (a and e and i and o and u) else "No")

File: problem-1/author-solutions/s.cpp
#include<iostream>
using namespace std;
int main(void) {
    string s; getline(cin, s);
    bool a, e, i, o, u;
    a = e = i = o = u = false;
    for(char c: s) {
      if (!a && tolower(c) == 'a') a = true;
      if (!e && tolower(c) == 'e') e = true;
      if (!i && tolower(c) == 'i') i = true;
      if (!o && tolower(c) == 'o') o = true;
      if (!u && tolower(c) == 'u') u = true;
    }
    cout << ((a && e && i && o && u) ? "Yes" : "No");
    return 0;
}